freshmaker/lightblue.py

```diff
@@ -228,6 +229,15 @@ def get_additional_data_from_koji(cls, nvr):
                         "Cannot find task_id or container_koji_task_id "
                         "in the Koji build %r" % build)
 
+            fs_koji_task_id = build.get('extra', {}).get('filesystem_koji_task_id')
+            if fs_koji_task_id:
+                parsed_nvr = koji.parse_NVR(nvr)
+                name_version = f'{parsed_nvr["name"]}-{parsed_nvr["version"]}'
+                if name_version not in conf.image_extra_repo:
+                    msg = (f'Build for {name_version} contains filesystem_koji_task_id, '
+                           f'but extra image repo is not specified.')
+                    raise ValidationError(msg)
```

Comment:
We need to catch this exception and update the image with default additional data when it happens, just like what we're doing for the `KojiLookupError`: https://github.com/redhat-exd-rebuilds/freshmaker/blob/6d0c059439ec2f7ad8235c391e94350c5c0eb437/freshmaker/lightblue.py#L302